charmcraft.yaml

```diff
@@ -17,3 +17,19 @@ parts:
       - curl
     override-pull: |
       curl -L -O https://github.com/canonical/nrpe_exporter/releases/latest/download/nrpe_exporter-${CRAFT_TARGET_ARCH}
+  vector:
+    plugin: dump
+    source: .
+    build-packages:
+      - curl
+      - tar
+    override-pull: |
+      VERSION=0.27.0
+      if [ $CRAFT_TARGET_ARCH == "amd64" ]; then
+        ARCH=x86_64-unknown-linux-musl
+      elif [ $CRAFT_TARGET_ARCH == "aarch64" ]; then
+        ARCH=aarch64-unknown-linux-musl
+      fi
+
+      VECTOR_URI="https://packages.timber.io/vector/${VERSION}/vector-${VERSION}-${ARCH}.tar.gz"
+      curl -sSfL ${VECTOR_URI} | tar xzf - -C . --strip-components=3 ./vector-${ARCH}/bin/vector
```

Comment:
Should we be concerned here about (in)visibility to CVE scanning?

Reply:
Considering the very narrow scope of what it actually does and that it's from DataDog, sort-of yes, but there's not a latest release.